stable/redis/templates/svc.yaml

```diff
@@ -24,6 +24,9 @@ spec:
   - name: redis
     port: 6379
     targetPort: redis
+  {{- if .Values.nodePort.enabled }}
```

Comment:
Could you not key this off `if .Values.serviceType == 'NodePort' and Values.nodePort.port` rather than having the extra value `nodePort.enabled` ?

I don't see a ton of value in allowing a nodeport to be set if the service type isn't set to `NodePort`.   I guess it would still be functional if set to `LoadBalancer`, but its usefulness would be very questionable ?

Reply:
+1

You can see here an example: https://github.com/kubernetes/charts/pull/3973/files

Reply:
@sunilganatra Please change as suggested.